routes/routes.js

```diff
@@ -28,15 +28,19 @@ router.get('/me', verifyToken, authCtrl.refreshToken, function(req, res) {
 router.get('/users/friends', verifyToken, authCtrl.refreshToken, userCtrl.getFriends);
 // Get user
 router.get('/users/:id', verifyToken, authCtrl.refreshToken, userCtrl.getUser);
-// Update user
-router.post('/users', verifyToken, authCtrl.refreshToken, userCtrl.updateUser);
```

Comment:
Kept this method and its route for now. Not all users will upgrade to use Api v2.